### PR TITLE
Documentation: Adding flag descriptions of unit test scripts

### PR DESCRIPTION
**Pull request by leemgs**
Let's append the meaning of the 'priority' and 'gcstresslevel' flag.
    The additional description of the 'priority' flag  is derived from
    ./coreclr/tests/dir.props. And, that of the 'gcstresslevel' flag is
    derived from ./coreclr/src/gc/sample/gcenv.h.
    
    Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

**Comment by leemgs** [comment]
@kouvel  , Could you take a look at 'gcstresslevel' and 'priority' flag?

\CC:  @janvorli ,  @BruceForstall , @ramarag

**Comment by leemgs** [comment]
* After PR: 
```
E:\dotnet\coreclr\tests>buildtest.cmd help
Invalid command-line argument: help

Usage:
    buildtest.cmd [option1] [option2] ...
All arguments are optional. Options are case-insensitive. The options are:

/? -? /h -h /help -help: view this message.
Build architecture: only x64 is currently allowed (default: x64).
Build type: one of Debug, Checked, Release (default: Debug).
Visual Studio version: one of VS2013 or VS2015 to force using a particular
    Visual Studio version (default: VS2015).
clean: force a clean build (default is to perform an incremental build).
CrossGen: enables the tests to run crossgen on the test executables before executing them.
msbuildargs ... : all arguments following this tag will be passed directly to msbuild.
priority <N> : specify a set of test that will be built and run, with priority N.
    0: Build only priority 0 cases as an essential testcases (default)
    1: Build all tests with CLRTestPriority 1 and 0
gcstresslevel <N> : specify the GCStress level the tests should run under.
    0: None                                1: GC on transitions to preemptive GC
    2: GC on every allowable JITed instr   4: GC on every allowable JITed instr
    8: GC on every allowable NGEN instr   16: GC only on a unique stack trace
sequential: force a non-parallel build (default is to build in parallel
    using all processors).
longgctests: Build tests so that runtests.cmd will do a long-running GC test.
gcsimulator: Build tests so that runtests.cmd will do a GCSimulator test run.
IlasmRoundTrip: enables ilasm round trip build and run of the tests before executing them.
verbose: enables detailed file logging for the msbuild tasks into the msbuild log file.

E:\dotnet\coreclr\tests>
```

**Comment by leemgs** [comment]
@jyoungyun , Take a look, please.

**Comment by ramarag** [comment]
@leemgs, the way of getting GC stress run on tests is changing with  https://github.com/dotnet/coreclr/pull/5261

I would advice to remove the the GC stress changes you have done in buildtest.cmd

**Comment by ramarag** [comment]
@leemgs: you could add GC stress details in runtest.cmd(.sh) once #5261 goes through

**Comment by leemgs** [comment]
@ramarag , I will move the description of gcstresslevel from buildtest.cmd to runtest.* according to #5261. Thank you for your comments.

**Comment by kouvel** [comment]
CC @wtgodbe @sergiy-k

**Comment by kouvel** [comment]
LGTM. In CLRTest.Execute.Bash.targets though, GCStressLevel is having the following command to run:
export complus_gcstress=$(GCStressLevel)

Unix is case-sensitive for env vars, so I don't think this will actually work. I'll file a separate issue.

**Comment by ramarag** [comment]
@kouvel RussKeldorph beat you to it, the issue has been filed in https://github.com/dotnet/coreclr/issues/5245

**Comment by kouvel** [comment]
Excellent :)

**Comment by kouvel** [comment]
I wonder though, if it would be better to just set the env vars before running tests. I assume this would put the command into the run scripts, but building tests takes a long time.

**Comment by ramarag** [comment]
@kouvel  that is exactly what  #5261 is trying to do, i have opened up issues for other GC and JIt in https://github.com/dotnet/coreclr/issues/5083 and https://github.com/dotnet/coreclr/issues/5082 respectively

**Comment by kouvel** [comment]
I see, thanks! I guess priority makes sense because you have to build those tests too.

**Comment by ramarag** [comment]
@leemgs https://github.com/dotnet/coreclr/pull/5261 has been merged do go ahead and make your changes

**Comment by leemgs** [comment]
> #5261 has been merged do go ahead and make your changes

@ramarag, Updated, Thanks.

File: tests/buildtest.cmd [hunk 1]
```diff
@@ -365,6 +365,9 @@ echo clean: force a clean build ^(default is to perform an incremental build^).
 echo CrossGen: enables the tests to run crossgen on the test executables before executing them. 
 echo msbuildargs ... : all arguments following this tag will be passed directly to msbuild.
 echo priority ^<N^> : specify a set of test that will be built and run, with priority N.
+echo     0: Build only priority 0 cases as essential testcases (default)
+echo     1: Build only priority 1 cases as additional testcases
```
Comment (ramarag) [inline tests/buildtest.cmd]: Actually 1 builds both pri 1 and pri 0 tests

Comment (ramarag) [inline tests/buildtest.cmd]: look at https://github.com/dotnet/coreclr/blob/master/tests/dir.props#L118

File: tests/runtest.cmd [hunk 2]
```diff
@@ -356,6 +356,9 @@ echo jitstressregs n    - Runs the tests with COMPlus_JitStressRegs=n
 echo jitminopts         - Runs the tests with COMPlus_JITMinOpts=1
 echo jitforcerelocs     - Runs the tests with COMPlus_ForceRelocs=1
 echo gcstresslevel n    - Runs the tests with COMPlus_GCStress=n
+echo     0: None                                1: GC on transitions to preemptive GC
+echo     2: GC on every allowable JITed instr   4: GC on every allowable JITed instr
```
Comment (ramarag) [inline tests/runtest.cmd]: i think you meant here 
 1,    // GC on all allocs and 'easy' places
2,    // GC on transitions to preemptive GC

File: tests/runtest.sh [hunk 3]
```diff
@@ -51,6 +51,10 @@ function print_usage {
     echo '  --jitstressregs=<n>              : Runs the tests with COMPlus_JitStressRegs=n'
     echo '  --jitminopts                     : Runs the tests with COMPlus_JITMinOpts=1'
     echo '  --jitforcerelocs                 : Runs the tests with COMPlus_ForceRelocs=1'
+    echo '  --gcstresslevel n                : Runs the tests with COMPlus_GCStress=n'
+    echo '    0: None                                1: GC on transitions to preemptive GC'
+    echo '    2: GC on every allowable JITed instr   4: GC on every allowable JITed instr'
```
Comment (ramarag) [inline tests/runtest.sh]: same as above

File: tests/runtest.cmd (hunk 2 above)
Comment (ramarag) [inline tests/runtest.cmd]: https://github.com/dotnet/coreclr/blob/master/src/gc/sample/gcenv.h#L121

File: tests/buildtest.cmd (hunk 1 above)
Comment (leemgs) [inline tests/buildtest.cmd]: > Actually 1 builds both pri 1 and pri 0 tests

Fixed. 

```
[./src/dir.targets ] 

 67   <PropertyGroup> 
 68     <BuildAllProjects Condition=" '$(BuildAllProjects)' == ''">false</BuildAllProjects>
 69     <_WillCLRTestProjectBuild Condition="'$(_WillCLRTestProjectBuild)' == ''">false</_WillCLRTestProjectBuild>
 70     <_WillCLRTestProjectBuild Condition="'$(BuildAllProjects)' != true">true</_WillCLRTestProjectBuild>
 71     <_WillCLRTestProjectBuild Condition="'$(DisableProjectBuild)' != true And '$(BuildAllProjects)' == true And '$(CLRTes    tPriority)' &lt;= '$(CLRTestPriorityToBuild)'">true</_WillCLRTestProjectBuild>                                           
 72   </PropertyGroup>

```

File: tests/runtest.sh (hunk 3 above)
Comment (leemgs) [inline tests/runtest.sh]: > same as above

Ooops. I missed the changes. Updated. Thanks.

**Comment by ramarag** [comment]
LGTM

**Pull request merged**